src/Database/Schema/MysqlSchema.php

```diff
@@ -112,22 +112,22 @@ protected function _convertColumn(string $column): array
             return ['type' => $typeName, 'length' => null, 'precision' => $length];
         }
 
-        if (($col === 'tinyint' && $length === 1) || $col === 'boolean') {
+        if ($col === 'tinyint' || $col === 'boolean') {
```

Comment:
Are you just testing stuff? I'd not be overly happy when all my `TINYINT` columns were suddenly treated as flags after a bugfix update 🤔

Reply:
yes, trying to find out what's different.